be/src/storage/olap_common.h

```diff
@@ -269,6 +270,7 @@ struct OlapReaderStatistics {
 };
 
 typedef uint32_t ColumnId;
+typedef int32_t ColumnUID;
```

Comment:
Will the ColumnUID be negative?

Reply:
invalid column uid is `-1`, same as `TabletColumn`.